backend/src/modules/player-profiles/core/PlayerProfilesModuleCore.ts

```diff
@@ -0,0 +1,30 @@
+import {ModuleCore} from "../../../shared/core/ModuleCore";
+import {FindAllPlayerProfiles} from "./application/query/FindAllPlayerProfiles";
+import {FindAllPlayerProfilesQueryHandler} from "./application/query/FindAllPlayerProfilesQueryHandler";
+import {PlayerProfilesRepository} from "./application/PlayerProfilesRepository";
+import {PlayerProfile} from "./application/command/PlayerProfile";
+import {PlayerProfileCommandHandler} from "./application/command/PlayerProfileCommandHandler";
+import {DomainEventPublisher} from "../../../shared/core/application/event/DomainEventBus";
+import {CurrentTimeProvider} from "../../../shared/core/CurrentTimeProvider";
+
+export function PlayerProfilesModuleCore(
```

Comment:
Wyglada to baaardzo dobrze :) Napisz testy. Zapisywania do repozytorium nie musisz robic w tym PR. Ale wtedy Ci Query nie zadziala bez tego.

Reply:
teraz jak już jest kod dobrze nazwany i poprawiony biorę się za testy :-)